index.bs

```diff
@@ -5679,39 +5674,34 @@ steps may throw an exception.
       <dt>If IsArray(|input|)</dt>
       <dd>
 
-        1. Let |len| be the ToLength(Get(|input|,
+        1. Let |len| be [=!=] ToLength(Get(|input|,
```

Comment:
Thanks re: needing ! for the Get.

This is in "if IsArray(input)" clause, i.e. we're dealing with an array exotic object. Is that sufficient?